x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -130,6 +138,39 @@ export const EndpointList = () => {
   const [showTransformFailedCallout, setShowTransformFailedCallout] = useState(false);
   const [shouldCheckTransforms, setShouldCheckTransforms] = useState(true);
 
+  // what to do about this type?
+  // need to make the back button conditional
+  const { state: routeState = {} } = useLocation<PolicyDetailsRouteState>();
```

Comment:
Not sure what you mean here. The `headerBackComponent` should be optional?

Reply:
sorry this was a todo for myself which i already figured out